frontend/components/commons/forms/questions-form/fields/multi-label/ClassifierAnnotationForFeedbackTask.component.vue

```diff
@@ -0,0 +1,267 @@
+        this.selectedLabels = this.appliedLabels;
+      }
+    },
+  },
+  computed: {
+    searchText: {
+      get: function () {
+        return this.idState.searchText;
+      },
+      set: function (newValue) {
+        this.idState.searchText = newValue;
+      },
+    },
+    selectedLabels: {
+      get: function () {
+        return this.idState.selectedLabels;
+      },
+      set: function (newValue) {
+        this.idState.selectedLabels = newValue;
+      },
+    },
+    shownLabels: {
+      get: function () {
+        return this.allowToShowAllLabels
+          ? this.labels.length
+          : this.idState.shownLabels;
+      },
+      set: function (newValue) {
+        this.idState.shownLabels = newValue;
+      },
+    },
+    visibleLabels() {
+      const numberOfSelectedLabels = this.filteredLabels.filter(
+        (l) => l.selected
+      ).length;
+      const availableNonSelected =
+        this.shownLabels < this.filteredLabels.length
+          ? this.shownLabels - numberOfSelectedLabels
+          : this.shownLabels;
+      let nonSelected = 0;
+      return this.filteredLabels.filter((l) => {
+        if (l.selected) {
+          return l;
+        } else {
+          if (nonSelected < availableNonSelected) {
+            nonSelected++;
+            return l;
+          }
+        }
+      });
+    },
+    filteredLabels() {
+      return this.labels.filter((label) =>
+        label.class.toLowerCase().match(this.searchText.toLowerCase())
+      );
+    },
+    appliedLabels() {
+      return this.labels.filter((l) => l.selected).map((label) => label.class);
+    },
+    labels() {
+      // Setup all record labels
+      const labels = Object.assign(
+        {},
+        ...this.inputLabels.map((label) => ({
+          [label]: { score: 0, selected: false },
+        }))
+      );
+      // Update info with annotated ones
+      this.annotationLabels.forEach((label) => {
+        labels[label.class] = {
+          score: 0,
+          selected: true,
+        };
+      });
+      // Update info with predicted ones
+      this.predictionLabels.forEach((label) => {
+        const currentLabel = labels[label.class] || label;
+        labels[label.class] = {
+          ...currentLabel,
+          score: label.score,
+        };
+      });
+      // Dict -> list
+      return Object.entries(labels).map(([key, value]) => {
+        return {
+          class: key,
+          ...value,
+        };
+      });
+    },
+    annotationLabels() {
+      return this.record.currentAnnotation?.labels || [];
+    },
+    predictionLabels() {
+      return this.record.prediction?.labels || [];
+    },
+    allowToShowAllLabels() {
+      return this.paginationSize === 1 || false;
+    },
+    predictedAs() {
+      return this.record.predicted_as;
+    },
+  },
+  mounted() {
+    this.selectedLabels = this.appliedLabels;
+  },
+  methods: {
+    updateLabels(labels) {
+      if (this.isMultiLabel || labels.length > 0) {
```

Comment:
we can remove isMultilabel references and prop because in the component is by default

Reply:
I am not sure yet, because maybe for singlelabel I will replace the previous I did in v0 of feedback task by some component which will reuse also the ClassifierAnnotation component